onnxruntime/python/tools/kernel_explorer/kernels/gemm_test.py

```diff
@@ -144,9 +145,9 @@ def test_gemm_tunable_bert_cases(dtype, size, transab):
     _test_gemm(getattr(ke, wrapper_name), dtype, *size, *transab)
 
 
-def profile_gemm_func(f, dtype, m, n, k):
-    a_shape = (m, k)
-    b_shape = (k, n)
+def profile_gemm_func(f, transa: bool, transb: bool, dtype: str, m: int, n: int, k: int):
```

Comment:
What is transa?

Reply:
Sorry about this, it is coming my local stacked commit. Rebased.

Reply:
BTW, transa and transb is a convention in GEMM, so the name is perserved for GEMM "guru".

Reply:
Good to know. Thanks!